### PR TITLE
ep4 init

### PR DESCRIPTION
**Pull request by nvk**

### CODE REVIEW

**Comment by jonnymarshall** [comment]
Are you ok with me merging my own commits in this repo @nkv ?

**Comment by nvk** [comment]
Let’s not merge until the show notes are final and date happens so that we can rename the TBD I. The file

File: 2022-08-tbd-Episode-04.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+### Get in touch with the pod
+- twitter.com/BitcoinReviewHQ
+- Telegram https://t.me/BitcoinReviewPod
+- Email bitcoinreview@coinkite.com
+
+### Vulnerability Disclosures
+
+### Software Releases and Project Updates
+- Sparrow 1.6.6 https://twitter.com/sparrowwallet/status/1555159885762990083
```
Comment (nvk) [inline 2022-08-tbd-Episode-04.md:30]: should have the details here and sub-bullets
ie
- Sparrow 1.6.6 [DATE]
    - Authentication via Auth47 & LNURL-auth links
    - Improved performance for very deep wallets
    - Change from notification txes is spent last
    - Copy labels from deposit UTXOs into badbank

**Comment by justinmoon** [comment]
MacOS key repeat https://apple.stackexchange.com/a/83923

**Pull request merged**